charts/rancher-logging/0.2.3/charts/fluentd/charts/fluentd-windows/templates/daemonset.yaml

```diff
@@ -36,7 +36,7 @@ spec:
     spec:
       serviceAccountName: {{ template "fluentd.fullname" . }}
       initContainers:
-      - name: init-{{ template "fluentd.fullname" . }}
+      - name: init-{{ template "fluentd.fullname" . }}-01
```

Comment:
Whats the actual purpose of these 2 init containers ? `01` and `02` don't explain.

Reply:
Two separate powershell commands need to be ran, so we can either write a PS1 script to run them in one container, or have an init container for each command.

The purpose of each one is to create a different directory path (akin to `mkdir -p path/to/desired/directory`).

Reply:
I'll rename them to be clear on what they do.

Reply:
Done

Reply:
Much better